sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -344,6 +344,36 @@ case class Join(
   }
 }
 
+/**
+ * Append data to an existing DataSourceV2 table.
+ */
+case class AppendData(
```

Comment:
How does this logical plan node map to the 8 operations outlined in your SPIP?

Reply:
This is `InsertInto`. Wenchen wanted to call it `Append` because that is more specific.

Reply:
I updated the SPIP doc to have both names.